README.md

```diff
@@ -4,65 +4,85 @@
-interact with OpenWhisk services. With this tool to download and configure, you are able to manage OpenWhisk services
-from the command line and automate them through scripts.
-
+The OpenWhisk Command Line Interface (OpenWhisk CLI) is a unified tool that
+provides a consistent interface to interact with OpenWhisk services. With this
+tool to download and configure, you are able to manage OpenWhisk services from
+the command line and automate them through scripts.
 
 # Where to download the binary of OpenWhisk CLI
 
-The OpenWhisk CLI is available on the release page: [click here to download](https://github.com/apache/incubator-openwhisk-cli/releases).
-We currently have binaries available for Linux, Mac OS and windows under amd64 architecture. You can download the
-binary, which fits your local environment.
+The OpenWhisk CLI is available on the release page: [click here to
+download](https://github.com/apache/incubator-openwhisk-cli/releases). We
+currently have binaries available for Linux, Mac OS and windows under amd64
+architecture. You can download the binary, which fits your local environment.
 
 
 # How to build the binary locally
 
-You can also choose to build the binaries locally from the source code with Go tool.
+The OpenWhisk CLI is written in the Go language.  You have two options to build
+the binary locally:
+
+1.  Compile in your logal Go environment,
+2.  Build using the packaged Gradle scripts (including the 'gogradle' plugin)
+
+## Build the binary with Go
 
-Make sure that you have Go installed [installing Go](https://golang.org/doc/install), and `$GOPATH` is defined [Go development environment](https://golang.org/doc/code.html).
+Make sure that you have Go installed [installing
+Go](https://golang.org/doc/install), and `$GOPATH` is defined [Go development
+environment](https://golang.org/doc/code.html).
 
-Then download the source code of the OpenWhisk CLI and the dependencies by typing:
+Then download the source code of the OpenWhisk CLI and the dependencies by
+typing:
 
 ```
 $ cd $GOPATH
 $ go get github.com/apache/incubator-openwhisk-cli
 ```
 
-
-## Build the binary with Go
-
-Open an terminal, go to the directory of OpenWhisk CLI home directory, and build the binary via the following command:
+Open an terminal, go to the directory of OpenWhisk CLI home directory, and build
+the binary via the following command:
 
 ```
 $ go build -o wsk
 ```
 
-If you would like to build the binary for a specific operating system, you may add the arguments GOOS and
-GOARCH into the Go build command. Since it is only applicable under amd64 architecture, you have to set GOARCH
-to amd64. GOOS can be set to "linux" "darwin" or "windows".
+If you would like to build the binary for a specific operating system, you may
+add the arguments GOOS and GOARCH into the Go build command. Since it is only
+applicable under amd64 architecture, you have to set GOARCH to amd64. GOOS can
```

Comment:
Is this right, can't we build for other architectures besides amd64?

Reply:
Good catch.  Deleted the 'specific architectures' sentence since it seems to function under any architecture Go supports.